jaq-std/src/std.jq

```diff
@@ -128,3 +128,7 @@ def gsub(re; f): sub(re; f; "g");
 
 # I/O
 def input: first(inputs);
+
+# Date
+def   todate:   todateiso8601;
+def fromdate: fromdateiso8601;
```

Comment:
I really like you how respect my "maximum overlap" formatting idea. Thanks a lot for your care to detail!

Reply:
It shows that you have studied what was there, you understood the principle behind it, and that you have adapted to it so your addition fits in nicely. I cannot emphasise enough how I appreciate this mindset of yours.